client/src/scenes/SettingsProfile/index.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import {
+    Grid,
+    Menu,
+    Header,
+    Divider,
+    Image,
+    Input,
+    Button,
+    TextArea,
+    Icon,
+    Segment,
+    Dropdown
+} from 'semantic-ui-react';
+import { Formik, Form, Field } from 'formik';
+import NOTES from './notes';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import PageHeader from '../../containers/Header';
+import { updateUserSettings } from '../../routines/routines';
+
+import styles from './styles.module.scss';
+import avatarPlaceholder from './avatar-placeholder.svg';
+
+class SettingsProfile extends Component {
+  renderField = ({ field }) => <Input fluid {...field} />;
+
+  renderDisabledField = ({ field }) => <Input disabled fluid {...field} />;
+
+  renderTextArea = ({ field }) => (
+      <TextArea {...field} rows="3" placeholder="Tell us a little bit about yourself" className={styles.textarea} />
+  );
+
+  onSubmit = (values, { setSubmitting }) => {
+      const { id, bio, company, location, name, imgUrl, url } = values;
+      const payload = { id, settings: { bio, company, location, name, imgUrl, url } };
+      this.props.updateUserSettings(payload);
+  };
+
+  onUploadPhoto = ev => {
+      console.warn('Upload photo!');
+  };
+
+  onRemovePhoto = ev => {
+      console.warn('Remove photo!');
+  };
+
+  imgEditBtn = (
+    <Button secondary compact size="tiny">
+        <Icon name="edit" />
+      Edit
+    </Button>
+  );
+
+  render() {
+      return (
+          <div>
+              <PageHeader />
+              <Divider hidden />
+              <Grid container>
+                  <Grid.Column computer={4} tablet={8} mobile={16}>
+                      <Menu vertical>
+                          <Menu.Item header>Personal settings</Menu.Item>
+                          <Menu.Item>Profile</Menu.Item>
+                      </Menu>
+                  </Grid.Column>
+                  <Grid.Column computer={12} tablet={16} mobile={16}>
+                      <Header as="h2">Public profile</Header>
+                      <Divider />
+                      <Grid>
+                          <Grid.Column computer={11} tablet={16} mobile={16}>
+                              <Formik initialValues={this.props.currentUser} onSubmit={this.onSubmit}>
+                                  {({ isSubmitting }) => (
+                                      <Form className="ui form">
+                                          <Header as="h4">Name</Header>
+                                          <Field type="text" name="name" render={this.renderField} />
```

Comment:
What about validation and errors?
https://jaredpalmer.com/formik/docs/api/errormessage

Reply:
It's about first, last etc. name
Not about username
No validation is needed here